test/evp_extra_test.c

```diff
@@ -688,6 +688,41 @@ static int test_EVP_DigestVerifyInit(void)
     return ret;
 }
 
+static int test_EVP_Digest(void)
```

Comment:
Maybe either add a comment or change the name of the test so that we know what this test was added for in the future..

Reply:
Comment added